odpf/optimus/plugins/v1beta1/dependency_resolver.proto

```diff
@@ -21,10 +21,17 @@ service DependencyResolverModService {
     rpc GenerateDependencies(GenerateDependenciesRequest) returns (GenerateDependenciesResponse);
 }
 
+message PluginJobConfigs {
```

Comment:
why do we need a segregation between secrets and regular configs, plugin implementation knows the difference

Reply:
Sure, will remove this